fdbserver/masterserver.actor.cpp

```diff
@@ -439,7 +439,7 @@ ACTOR Future<Void> newSeedServers(Reference<MasterData> self,
 	state int idx = 0;
 	state std::map<Optional<Value>, Tag> dcId_tags;
 	state int8_t nextLocality = 0;
-	state UID teamId;
+	state UID seedTeamId = nondeterministicRandom()->randomUniqueID();
```

Comment:
Why not `deterministicRandom`?

Reply:
Yes, I meant it to be deterministic random, but my hands typed non!